package.json

```
@@ -55,6 +55,7 @@
     "gulp-terser": "2.1.0",
     "minimist": "1.2.6",
     "node-fetch": "2.6.7",
+    "playwright": "1.29.1",
```

Biome (1.9.4)

[error] The key playwright was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)